team.md

```diff
@@ -6,8 +6,6 @@ redirect_from:
 
 ## Who we are
 
-Red56 are a team of business-focused technologists based mostly in London.
```

Comment:
Yes, actually this was deliberate! (this is an expansion of the home page -- might get there from some other route).